### PR TITLE
Bulgarian Translation

### PR DESCRIPTION
**Pull request by kalinchernev**
Initially translated by me and later validated and kindly corrected by @ivaylospasov.
History of commits contain further details if necessary.
Feedback is welcome! :)

### CODE REVIEW

**Comment by martinheidegger** [comment]
The team effort is awesome 👍 👍 👍 
@riyadhalnur Seems like they already did peer review (looking at the commits)?!

**Comment by jpwesselink** [comment]
@lulusam

**Comment by kalinchernev** [comment]
Hi all, I asked @ivaylospasov for a review before opening the pull request as he's working at a national media in Bulgaria and have perfect control of the written Bulgarian.
Further feedback is welcome! :)

**Comment by martinheidegger** [comment]
@riyadhalnur Do you think the current review status is okay? @lulusam do you have time for a review?

**Comment by jpwesselink** [comment]
Yes sorry, @lulusam, me mentioning you from mobile device lacked context, my apologies. Could you please take a peek at the Bulgarian translations? :D

**Comment by lulusam** [comment]
Hello guys, amazing effort! @jpwesselink no problem, thank you for the mention never the less! @martinheidegger indeed I would love to go trough the translations!:)

File: languages/bg.json [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  "about-info2": "За организацията на NodeSchool ние използваме GitHub. Най-доброто място, на което можете да се свържете с нас, е <a href=\"https://github.com/nodeschool/organizers/issues\">хранилището за организатори</a>. Докладвате за нов проблем тук, организаторите ще бъдат известени и ще могат да отговорят. Този метод е много по-добър, отколкото изпращане на директен имейл до който и да е от организаторите.",
+  "about-header-talk-title": "Чуй и виж",
+  "about-info3": "В <a href=\"https://archive.org/details/NodeUp55\" target=\"_blank\">епизод 55</a> на подкаста \"NodeUp\" Mikeal Rogers, Max Ogden и други членове на общността говорят за NodeSchools. На \"Cascadia JS 2014\" Jason Rhodes от Балтимор, <a href=\"https://www.youtube.com/watch?v=YJ7txKTh3-E\" target=\"_blank\">говори за това как се управлява NodeSchool</a>.",
+  "footer-contact-header": "Контакт",
+  "footer-contribute-header": "Допринеси",
+  "footer-contribute-question": "Докладвай за проблем",
+  "footer-contribute-answer": "Отговори на въпрос",
+  "footer-about-header": "Относно",
+  "footer-about-build": "Създай упражнение",
+  "footer-about-host": "Организирай работилница",
```
Comment (lulusam) [inline languages/bg.json]: This confuses workshopper with workshop.
screen shot 2016-09-20 at 08 16 21

File: languages/bg.json [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  "hexdex-h1": "Hexdex",
+  "hexdex-header": "Списък с <a href=\"chapters.html\">клонове на NodeSchool клонове</a>, изобразени чрез шестоъгълници.",
+  "hexdex-make-logo": "Направете един и за своя клон, като използвате <a href=\"https://github.com/nodeschool/nodeschool.github.io/tree/master/images/make-a-sticker\">този темплейт</a>.",
+  "building-workshops-header": "Създаване на NodeSchool работилница",
+  "building-workshops-subtitle": "Ето някои от популярните инструменти, които се използват за направата на NodeSchool работилници.",
+  "building-workshops-title-info": "<a href=\"https://www.npmjs.org/package/workshopper\">Упражнение</a>",
+  "building-workshops-info": "за да вземете напълно функционален NodeSchool framework за създаване упражнение с CLI.",
+  "building-workshops-title-info2": "<a href=\"https://www.npmjs.org/package/adventure\">Приключение</a>",
+  "building-workshops-info2": "за да вземете по-лека алтернатива на Упражнението.",
```
Comment (lulusam) [inline languages/bg.json]: I suggest sticking to workshopper instead of exercise for the entirety of the text.

File: languages/bg.json [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+  "index-header-bottom": "Стартирайте, инсталирайки някой от нашите",
+  "index-header-bottomA": "основни работилници",
+  "index-header-bottomB": "или се запишете за нашия безплатен",
+  "index-header-bottomC": "имейл бюлетин",
+  "index-workshop-header": "Работилници",
+  "index-workshop-info": "Работилниците, провеждани на живо, обикновено са безплатни. Упражненията в тях се използват като учебна програма, а ментори помагат на участниците в решаването на предизвикателствата.",
+  "index-workshop-links-host": "Организирайте събитие",
+  "index-workshop-links-events": "Вижте всички събития",
+  "index-workshop-links-writeups": "Обобщение от предишни събития",
+  "index-upcoming-header": "Предстоящи работилници",
```
Comment (lulusam) [inline languages/bg.json]: събития instead of работилници

File: languages/bg.json [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+  "events-small": "Ако трябва да редактирате вече публикувано събитие, <a href=\"https://github.com/nodeschool/organizers/issues/new\">пишете ни в хранилището nodeschool/organizers.</a>",
+  "calendar-h1": "Календар",
+  "index-header-top": "Работилници с отворен код, които обучават на софтуерни умения за уеб. Можете да ги правите самостоятелно или на някое събитие близо до Вас.",
+  "menu-tutorials": "Уроци",
+  "index-header-bottom": "Стартирайте, инсталирайки някой от нашите",
+  "index-header-bottomA": "основни работилници",
+  "index-header-bottomB": "или се запишете за нашия безплатен",
+  "index-header-bottomC": "имейл бюлетин",
+  "index-workshop-header": "Работилници",
+  "index-workshop-info": "Работилниците, провеждани на живо, обикновено са безплатни. Упражненията в тях се използват като учебна програма, а ментори помагат на участниците в решаването на предизвикателствата.",
```
Comment (lulusam) [inline languages/bg.json]: Oбучения instead of Работилници

File: languages/bg.json [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+  "index-upcoming-all-events": "Вижте всички събития",
+  "index-upcoming-error": "Съжаляваме, но събитията не могат да бъдат заредени!",
+  "index-host-header": "Всеки може да организира събитие",
+  "index-host-past": "До този момент общият брой е ",
+  "index-host-past2": "събития!",
+  "index-host-want-to-host": "NodeSchool е децентрализиран проект с отворен код, управляван от доброволци. Искате ли да организирате събитие?",
+  "index-host-want-to-host2": "Източници и съвети",
+  "index-host-want-to-host3": "с които да разберете как става това.",
+  "index-workshopper-header": "Упражнения",
+  "index-workshopper-info": "Упражнение е име, използвано при модулите с отворен код в NodeSchool. Всички те са подходящи за самообучение (не е задължително да присъствате на събитие, за да работите по някой от тях), а и повечето работят офлайн.",
```
Comment (lulusam) [inline languages/bg.json]: Работилници instaed of Упражнения

File: languages/bg.json [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+  "mentoring-list-expected-3": "Мотивирайте и окуражавайте участниците. Покажете разбиране към техните проблеми.",
+  "mentoring-list-not-expected-headline": "Какво <em>НЕ &#xA0;</em> се очаква от Вас?",
+  "mentoring-list-not-expected-1": "Да говорите пред групата.",
+  "mentoring-list-not-expected-2": "Да подготвите всички материали и решенията им.",
+  "mentoring-list-not-expected-3": "Да имате отговор на абсолютно всеки въпрос.",
+  "mentoring-still-unsure": "<h2 data-i18n=\"mentoring-still-unsure-headline\">Все още не сте сигурни дали можете да се справите?</h2>\n          <p data-i18n=\"mentoring-still-unsure-part1\">\n            Просто опитай някой от <a href=\"/#workshopper-list\">основните workshopper-и</a>.\n          </p>",
+  "mentoring-still-unsure-headline": "Все още не сте сигурни дали можете да се справите?",
+  "mentoring-still-unsure-part1": "Просто опитайте някой от <a href=\"/#workshopper-list\">основните упражнения</a>.",
+  "host-header": "Организиране на NodeSchool събития",
+  "host-info": "NodeSchool е усправляван от общността проект със 100% отворен код. Това означава, че всеки може да ползва материали от NodeSchool и/или да организира събитие.",
```
Comment (lulusam) [inline languages/bg.json]: усправляван -> управляван

**Comment by martinheidegger** [comment]
@lulusam Awesome!

**Comment by riyadhalnur** [comment]
Thank you @lulusam for going over it 😄 I think once the changes are made, we'll be good to go

**Comment by lulusam** [comment]
My pleasure! Very excited to see the website in the mother language:D

File: languages/bg.json (hunk 1 above)
Comment (kalinchernev) [inline languages/bg.json]: Do you mean: "Организирай събитие" instead of "Организирай работилница"?

**Comment by kalinchernev** [comment]
@lulusam comments and suggestions applied - many thanks for your time to make the review!
I'd like to still make a test with the strings on a build soon, please hold on with the merge for a while.

Also, just want to double check understanding:
- workshopper is to be changed instead of упражнение everywhere?
- Any specific rule about where to change работилница to събитие? Or the rest of the instances of работилница as used currently are ok?

Many thanks again!

**Comment by kalinchernev** [comment]
Hi @lulusam the current version of the translations should be ok.
Any further feedback is welcome :)

**Comment by riyadhalnur** [comment]
Hi @lulusam, did you get a chance to look over the changes @kalinchernev made?

**Comment by lulusam** [comment]
@riyadhalnur @kalinchernev , hey guys, I am sorry I have missed the previous message! The changes look great! Greets, Lily

**Comment by riyadhalnur** [comment]
@lulusam No worries. I'm taking liberty then of merging this in. Thank you for the hard work @kalinchernev, @ivaylospasov and @lulusam 😃

**Comment by kalinchernev** [comment]
Thanks a lot everyone! :)

**Pull request merged**